lib/react_client.dart

```diff
@@ -181,6 +191,8 @@ class ReactDartComponentFactoryProxy2<TComponent extends Component2> extends Rea
   /// Returns a JavaScript version of the specified [props], preprocessed for consumption by ReactJS and prepared for
   /// consumption by the [react] library internals.
   static JsMap generateExtendedJsProps(Map props) {
```

Comment:
This was previously a static member on `ReactDartComponentFactoryProxy2`; will moving it into this class break over_react? We might just want to proxy it on that class for that reason, and plus, it'd be a little more convenient to get it from there instead of knowing to go get it from the mixin.